### DISCUSSION TITLE
Add search functionallity

### DISCUSSION THREAD

**Discussion by PaulWoitaschek** [PaulWoitaschek/Voice#1304] [Ideas]
See #689

**Comment by Celebrimbor-forge** [comment]
nice

**Comment by Celebrimbor-forge** [comment]
Add another 'assignment' readme example

**Comment by PaulWoitaschek** [comment]
> Add another 'assignment' readme example

What does this mean?

**Comment by thw0rted** [comment]
Duplicate of #571 , which has more detail. Maybe close this one?
